jujugui/static/gui/src/app/components/panel/_panel.scss

```diff
@@ -25,4 +25,5 @@
     background-color: rgba(255, 255, 255, 0.3);
     border-radius: none;
     box-shadow: none;
+    overflow: scroll;
```

Comment:
I tried adding it to a child and it doesn't work properly; the search results end up on top of the commit bar. It doesn't seem like this rule should break panels in general, but rather just control scroll for any .white-box content, which seems like what we want.